package.json

```diff
@@ -87,6 +89,7 @@
     "jest": "^25.2.7",
     "json-comb": "^0.2.43",
     "nodemon": "^2.0.2",
+    "patch-package": "^6.2.0",
```

Comment:
need this for the route generator (see my note above)

Reply:
oops, my bad. I removed it because I thought it was related to the webpack config that we weren't using.